src/lib.rs

```diff
@@ -13,6 +13,25 @@
 //!    println!("Status: {} {}", res.status_code(), res.reason());
 //!}
 //!```
+#![cfg_attr(all(feature = "sgx", not(target_env = "sgx")), no_std)]
+#![cfg_attr(target_env = "sgx", feature(rustc_private))]
```

Comment:
I'm not sure if we need these, in our own libraries, we just have:
`#![cfg_attr(not(feature = "std"), no_std)]` 

But if it works with these attributes defined, then we can leave them I think.

Reply:
#![cfg_attr(all(feature = "sgx", not(target_env = "sgx")), no_std)] is required, else the teeracle enclave runtime doesn't compile: duplicate lang item in crate `sgx_tstd` (which `http_req` depends on)
#![cfg_attr(target_env = "sgx", feature(rustc_private))], I don't know.